sigar_interface.go

```diff
@@ -353,19 +353,26 @@ type ProcIo struct {
 }
 
 type ProcMem struct {
-	Size        uint64
-	Resident    uint64
-	Share       uint64
-	MinorFaults uint64
-	MajorFaults uint64
-	PageFaults  uint64
+	Size          uint64
+	Resident      uint64
+	Share         uint64
+	MinorFaults   uint64
+	MajorFaults   uint64
+	PageFaults    uint64
+	PageFileBytes uint64
 }
 
 type ProcTime struct {
-	StartTime uint64
-	User      uint64
-	Sys       uint64
-	Total     uint64
+	CollectionTime time.Time
+	StartTime      uint64
+	User           uint64
+	Sys            uint64
+	Total          uint64
+
+	// Not valid until after Diff()
+	PercentUserTime  uint64
+	PercentSysTime   uint64
+	PercentTotalTime uint64
```

Comment:
They can go above 100 if a process is using multiple threads. 8-bit is probably too low, but 32-bit should suffice. I decided to leave them at 64-bit since almost all other sigar values are 64-bit, and they will likely be cast to 64-bit for display anyway.